src/main/java/com/example/simpleannotation/processors/AbstractAnnotationProcessor.java

```diff
@@ -32,16 +34,16 @@ public boolean process(Set<? extends TypeElement> annotations,
                     M transformedModel = transformElementToModel(annotatedElement,
                             annotatedElement.getAnnotation(annotationClazz));
                     finalizeElementProcessing(transformedModel);
-                } catch (BadAnnotationUsageException baue) {
-                    printErrorMessage(baue.getMessage(), annotatedElement);
+                } catch (BadAnnotationUsageException | ClassGenerationException e) {
+                    printErrorMessage(e.getMessage(), annotatedElement);
                 }
         }
-        return true;
+        return false;
     }
 
     private void printErrorMessage (String errorMessage, Element element) {
```

Comment:
delete extra space here and elsewhere

Reply:
Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)